tools/statistical_ensemble_test/single_run.py

```diff
@@ -313,6 +325,10 @@ def single_case(opts_dict, case_flags, stat_dir):
             command = "./xmlchange --file env_run.xml --id STOP_N --val 12"
             ret = os.system(command)
 
+    print("STATUS: updating ROF_NCPL...")
```

Comment:
This should not be needed here - what prompted it?

Reply:
Aha, I just leave it here for sanity check. I could remove it in my future update of ECT.

Reply:
I actually was referring to line 329.  Why is it required now - it wasn't needed before this?

Reply:
Oh, sorry that I misunderstood your question. This line is need for the NUOPC driver only in a nine-time-step simulation. Otherwise I will get the error message `"Invalid argument  - setClock timeStep=10800s is not a divisor of runDuration=5400s". I think it is related to the ESMF settings.